packages/xo-server-audit/src/index.js

```diff
@@ -108,6 +109,9 @@ class AuditXoPlugin {
     }
     this._cleaners = []
     this._xo = xo
+    this._automaticIntegrityCheckJob = createSchedule(
+      staticConfig.automaticIntegrityCheckCron ?? '0 6 * * *'
+    ).createJob(() => this._automaticIntegrityCheck().catch(log.error))
```

Comment:
Which timezone is used for this schedule?

Maybe it should be made configurable as well?